packages/kbn-optimizer/limits.yml

```diff
@@ -110,7 +110,7 @@ pageLoadAssetSize:
   navigation: 37269
   newsfeed: 42228
   noDataPage: 5000
-  observability: 76678
+  observability: 118191
```

Comment:
is this needed?

Reply:
yeah i am exporting a hook, i have washed my sins here :D 

https://github.com/elastic/kibana/pull/186820